src/libraries/Microsoft.Extensions.Logging/tests/DI.Common/Common/src/TestLogger.cs

```diff
@@ -41,11 +41,6 @@ public IDisposable BeginScope<TState>(TState state)
 
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
-            if (!IsEnabled(logLevel))
```

Comment:
would this not break any existing test?

Reply:
No existing tests broke.

The reason for that change, is to see if `skipEnabledCheck` works as https://github.com/dotnet/runtime/blob/311d37de4750d856f3e2292fdf8bacc1bcc62709/src/libraries/Microsoft.Extensions.Logging/tests/Common/LoggerMessageTest.cs#L202 shouldn't be logged if set to `false`, but should be logged when set to `true`. 
If `Log`-method would check, then it won't be logged, thus it's removed.

A different approach would be to call-count `IsEnabled` and assert that value.

Reply:
I see, I think call-counting `IsEnabled` and asserting sounds a bit clearer for future reference.

Reply:
Changed to call-counting -> d1b4e93652016f93eceafcadbe1ac1f0b0ae3de5